data/json/recipes/weapon/explosive.json

```diff
@@ -28,8 +28,9 @@
     "tools": [ [ [ "hotplate", 25 ], [ "toolset", 25 ], [ "fire", -1 ] ] ],
     "components": [
       [ [ "paper", 5 ], [ "rolling_paper", 32 ] ],
-      [ [ "acid", 2 ] ],
-      [ [ "ammonia", 1 ] ],
+      [ [ "acid", 1 ], [ "chem_sulphuric_acid", 1 ] ],
+      [ [ "chem_nitric_acid", 1 ] ],
```

Comment:
Nitric acid is too hard to get, this makes this part of the recipe too limiting.
Would be better to just allow 2 of any acid.